src/nft/nftId.ts

```diff
@@ -0,0 +1,25 @@
+import { createHash } from 'crypto';
+import { bech32 } from 'bech32';
+import { getISCNPrefix } from '../iscn/parsing';
+
+export function calculateNFTClassIdByAccount(
```

Comment:
Should rename to `calculateNFTClassId`.

In chain code there are 2 functions: `NewClassIdForAccount` and `NewClassIdForISCN`.
The first one calculate class ID for the case of parent address, which uses the raw bytes of the address (the 20 bytes behind Bech32 encoding).
The second one uses string ISCN ID prefix.
Both of them calls an internal function called `newClassId`.

In this PR, `calculateNFTClassIdByISCNId` corresponds to `NewClassIdForISCN` in chain code.
So to make it more consistent, we should use something like `calculateNFTClassId` to correspond to `newClassId` in chain code, and leave `calculateNFTClassIdByAccount` for `NewClassIdForAccount` in the future.

Also, in chain code `newClassId` takes bytes instead of string, since `NewClassIdForAccount` already has the bytes.